src/components/Form/Form.css

```diff
@@ -1,39 +1,116 @@
+@font-face {
+  font-family: "Poppins";
+  src: url("../../Fonts/Poppins-SemiBold.ttf");
+}
+
+@font-face {
+  font-family: "Poppins_bolder";
+  src: url("../../Fonts/Poppins-Bold.ttf");
+}
+
+@font-face {
+  font-family: "Poppins_light";
+  src: url("../../Fonts/Poppins-ExtraLight.ttf");
+}
+
 .Background_Form {
   height: 100vh;
   background: linear-gradient(154deg, #ff5757 0%, hsl(338, 100%, 50%) 100%);
+  font-family: "Poppins";
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
 }
 
-.Trott_Logo {
-  margin: 0;
+.TrottForm_Logo {
+  width: 60vw;
   color: white;
-  font-size: 3rem;
 }
 
-.Enter_Number {
-  color: white;
-  font-size: 1.5rem;
+.Text_Container {
+  width: 85vw;
+  margin-bottom: 2rem;
 }
 
-.Input_tel {
-  margin: 3rem;
+.Enter_Number {
+  font-family: "Poppins_bolder";
+  font-size: 1.5rem;
+  text-align: start;
+  color: white;
 }
 
 .Text_Form {
   color: white;
+  text-align: start;
+  font-size: 0.95rem;
 }
 
+.Input_Container {
+  width: 90vw;
+}
 
-button{
-  margin: 1rem;
+.Input_Phone {
+  border: none;
+  background-color: transparent;
+  color: white;
 }
 
-.Without_Connect {
+::-webkit-input-placeholder {
+  color: rgb(255, 255, 255);
+
+.Text_Form_Align {
+  font-family: "Poppins_light";
```

Comment:
replace with:

    font-weight: 200;

(this the light weight)

Reply:
done, I use 500 as value